code/datums/speech/accents.dm

```diff
@@ -0,0 +1,191 @@
+TYPEINFO(/datum/speech_module/accent/chav)
+	id = "accent_chav"
+/datum/speech_module/accent/chav
+	id = "accent_chav"
+
+	process(datum/say_message/message)
+		var/mob/speaker = message.speaker
+		if(!istype(speaker))
```

Comment:
do we need to be checking the speaker is a mob here? Can't think of a reason why an accent couldn't also be applied to any `/atom`